CONTRIBUTING.md

```diff
@@ -35,8 +35,47 @@ just a few small guidelines you need to follow.
         1.  For Google-authored source files, paste the Apache header text in
             the comments at the top. (If you're using different license,
             include the full text of that license.)
+1.  Place an executable called `pipeline` on the root of your solution folder
```

Comment:
we could include an example...

specifically, this must include a header at the top such as `#!/bin/sh` or `#!/usr/bin/env node` else travis will not know how to interpret this.

this may seem like a trivial point, but if contributors write nodejs or python without this heading, then it won't work

Reply:
Not sure that is correct. We already have an example in hello-world and it doesn't include any such headers. We only need an "executable". Teaching what that means seems a little overkill, don't you think?

Reply:
that is just because bash is the default. if i write node without this header, i get an error...

```
$ docker run -it alpine
/ # echo 'console.log("test")' > pipeline
/ # sh pipeline
pipeline: line 1: syntax error: unexpected word (expecting ")")
```

true - teaching this may be overkill. but at least a link to useful docs? otherwise i am sure contributors will face this

Reply:
Something like..

`If you have any problems, please make sure you are using the appropriate she-[bang](https://en.wikipedia.org/wiki/Shebang_(Unix)#Examples)`